core/models.py

```diff
@@ -226,6 +234,23 @@ class Invitation(models.Model):
         'Circle',
         related_name='+',
     )
+    DEFAULT_EXPIRATION = timedelta(days=7)
+
+    is_open = models.BooleanField(default=False)
+    expires_at = models.DateTimeField(null=True, blank=True)
+
+    def is_expired(self):
+        if not self.is_open and self.expires_at:
+            return timezone.now() > self.expires_at
+        return False
+
+    @property
```

Comment:
I don't remember why I added the other stuff. 
Do you agree with the 
is_open attribute?
Correct expiration approach?